pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, mkDerivation, fetchFromGitHub,
+cmake, pkgconfig, glew, opencv3, libusb, libopus,
+boost, qtbase, qtwebsockets, qtmultimedia, wrapQtAppsHook,
+fftw, fftwFloat,
+limesuite,
+ocl-icd,
+libpulseaudio, libav, libiio
+}:
+
+let
+
+  codec2 = stdenv.mkDerivation {
+    name = "codec2";
+    src = fetchFromGitHub {
+      owner = "drowe67";
+      repo = "codec2";
+      rev = "567346818c0d4d697773cf66d925fdb031e15668";
+      sha256 = "0ngqlh2cw5grx2lg7xj8baz6p55gfhq4caggxkb4pxlg817pwbpa";
+    };
+    nativeBuildInputs = [ cmake ];
+  };
+
+  cm256cc = stdenv.mkDerivation {
+    name = "cm256cc";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "cm256cc";
+      rev = "f21e8bc1e9afdb0b28672743dcec111aec1d32d9";
+      sha256 = "1imxg9hxh92r8ip161980axavvx1sr321lw0g8rbza59s50sd17g";
+    };
+    nativeBuildInputs = [ cmake ];
+    buildInputs = [ boost ];
+  };
+
+  serialdv = stdenv.mkDerivation {
+    name = "serialdv";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "serialdv";
+      rev = "d5830fae715f4acffa2acec33539c3a11c17a1c9";
+      sha256 = "0swalyp8cbs7if6gxbcl7wf83ml8ch3k7ww4hws89rzpjvf070fr";
+    };
+    nativeBuildInputs = [ cmake ];
+  };
+
+in mkDerivation rec {
+  pname = "sdrangel";
+  version = "4.11.6";
+  src = fetchFromGitHub {
+    owner = "f4exb";
+    repo = "sdrangel";
+    rev = "ec22056cbf43f1150c610fd14df447cdadeadd45";
+    sha256 = "05vijflnydl10l0h7ib6pwi06fmg6k740vcbkm0g5293s2z85j14";
+    fetchSubmodules = false;
+  };
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [
+    glew opencv3 libusb boost libopus limesuite libav libiio libpulseaudio
+    qtbase qtwebsockets qtmultimedia wrapQtAppsHook
+    fftw fftwFloat
+    codec2 cm256cc serialdv
+  ];
+  cmakeFlags = [
+    "-DLIBSERIALDV_INCLUDE_DIR:PATH=${serialdv}/include/serialdv"
+    "-DLIMESUITE_INCLUDE_DIR:PATH=${limesuite}/include"
+    "-DLIMESUITE_LIBRARY:FILEPATH=${limesuite}/lib/libLimeSuite.so"
+  ];
+
+  LD_LIBRARY_PATH = "${ocl-icd}/lib";
+
+  meta = with stdenv.lib; {
+    description = "Software defined radio (SDR) software";
+    longDescription = ''
+        SDRangel is an Open Source Qt5 / OpenGL 3.0+ SDR and signal analyzer frontend to various hardware.
+    '';
+    homepage = https://github.com/f4exb/sdrangel;
```

Comment:
Please quote the URL as unquoted URLs are deprecated.

Reply:
I just copied, i haven't seen a single package that does that, looked weird to me but i just did like the others ^

Reply:
Makes sense. I'm just trying to help the maintainers in the future that will need to quote all non quoted URLs when they'll want to finally drop the support for this.

Reply:
Oh ok then ^